source/images/supported_brands/QuantitySoldBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
What is the purpose of this file?

Reply:
Whoops, must have gotten accidentally dragged in there from somewhere else. Deleted.